#errors/exception handling
a=[1,2,3,4]
try:
    print(a[9])
except:
    print("check the array size")
print("thank you")
